docs/server/mcp/tools/get-example.ts

```
@@ -2,18 +2,28 @@ import { z } from 'zod'
+    { exampleName: 'ButtonBasic' },
+    { exampleName: 'ModalOverlay' }
+  ],
   cache: '30m',
   async handler({ exampleName }) {
+    let result
     try {
-      const result = await $fetch<{ code: string }>(`/api/component-example/${exampleName}.json`)
-      return {
-        content: [{ type: 'text' as const, text: result.code }]
-      }
+      result = await $fetch<{ code: string }>(`/api/component-example/${exampleName}.json`)
```

ESLint

[error] Operator '<' must be spaced.

(`@stylistic/space-infix-ops`)

---

[error] 'string' is not defined.

(no-undef)